--- lesstif-0.93.94/scripts/autoconf/ac_find_motif.m4.orig	2001-09-25 22:40:58.000000000 +0200
+++ lesstif-0.93.94/scripts/autoconf/ac_find_motif.m4	2003-12-29 14:02:58.877412536 +0100
@@ -18,7 +18,7 @@
 dnl Oleo CVS Id: motif.m4,v 1.9 1999/04/09 11:46:49 danny
 dnl LessTif CVS $Id$
 dnl
-AC_DEFUN(AC_FIND_MOTIF,
+AC_DEFUN([AC_FIND_MOTIF],
 [
 
 AC_REQUIRE([AC_PATH_XTRA])
@@ -220,7 +220,7 @@
 dnl Makefiles. Perhaps one should immediately add those libs
 dnl to link commands which include libXm version2.1?!
 dnl
-AC_DEFUN(AC_FIND_LIBXP,
+AC_DEFUN([AC_FIND_LIBXP],
 [AC_REQUIRE([AC_PATH_X])
 AC_CACHE_CHECK(whether libXp is available, lt_cv_libxp,
 [lt_save_CFLAGS="$CFLAGS"
