--- lesstif-0.93.94/lib/Xm-2.1/Makefile.am.orig	2001-08-29 18:06:48.000000000 +0200
+++ lesstif-0.93.94/lib/Xm-2.1/Makefile.am	2003-12-29 18:36:40.446954504 +0100
@@ -21,7 +21,7 @@
 XDND_SRC=
 endif
 
-libXm_la_LIBADD = $(X_PRE_LIBS) -lXt $(XPLIB) $(X_LIBS) -lX11 $(X_EXTRA_LIBS)
+libXm_la_LIBADD = $(X_PRE_LIBS) -lXt $(XPLIB) $(X_LIBS) -lX11 $(X_EXTRA_LIBS) $(XFT_LIBS)
 
 #if Have_Libxp
 #libXm_la_LIBADD = -lXt $(X_PRE_LIBS) -lXp -lXext -lX11 \
--- lesstif-0.93.95/configure.in.orig	2004-08-06 12:10:23.000000000 +0200
+++ lesstif-0.93.95/configure.in	2004-08-07 16:28:41.694547168 +0200
@@ -466,7 +466,12 @@
 dnl
 dnl Experimental ! Keith Packard's Xft
 dnl
+prexft_save_LIBS="$LIBS"
+LIBS=
 AC_FIND_XFT
+XFT_LIBS="$LIBS"
+LIBS="$prexft_save_LIBS"
+AC_SUBST(XFT_LIBS)
 
 dnl
 dnl Now ensure that all required variables get substituted!
