diff -ur lesstif-0.95.0.orig/lib/Dt/Makefile.am lesstif-0.95.0/lib/Dt/Makefile.am
--- lesstif-0.95.0.orig/lib/Dt/Makefile.am	2007-11-23 21:38:17.000000000 +0100
+++ lesstif-0.95.0/lib/Dt/Makefile.am	2007-11-23 21:38:26.000000000 +0100
@@ -5,7 +5,7 @@
 MAINTAINERCLEANFILES=Makefile.in
 
 libDtPrint_la_LDFLAGS=	-version-info 1:0 -no-undefined
-libdir = $(exec_prefix)/lib
+#libdir = $(exec_prefix)/lib
 
 if BuildLibDtPrint
 
